docs/modules/ROOT/pages/highlights.adoc

```diff
@@ -0,0 +1,9 @@
+= What's new in Play SOAP 2.x
+
+This page highlights the new features of Play SOAP 2.x.
```

Comment:
I would just remove this, personal preference.

Reply:
Done ✔️